docs/source/frontends/jfrontend.rst

```diff
@@ -6,8 +6,6 @@ Jfrontend
 Environments
 ############
 
-Change renderer
```

Comment:
It becomes redundant. Hence, removed "Change renderer" because the heading is already provided below each image.